modules/ROOT/pages/installation/osx.adoc

```diff
@@ -43,8 +43,8 @@ If the password is not set explicitly using this method, it will be set to the d
 In that case, you will be prompted to change the default password at first login. +
 For more information, see xref:configuration/set-initial-password.adoc[].
 . Start Neo4j:
-* To run Neo4j as a console application, use: `$NEO4J_HOME/bin/neo4j console`.
-* To run Neo4j in a background process, use: `$NEO4J_HOME/bin/neo4j start`.
+* To run Neo4j as a console application, use: `$NEO4J_HOME/bin/neo4j server console`.
```

Comment:
Actually, `neo4j server console` is wrong. You can either use `$NEO4J_HOME/bin/neo4j console` or `$NEO4J_HOME/bin/neo4j-admin server console`. The same applies to `neo4j start`. You can find more information, [here](https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/#_the_neo4j_tool).